### ISSUE TITLE
Add aria-haspopup support notes

### ISSUE THREAD

**Issue by tesk9** [tesk9/accessible-html-with-css#26]
`aria-haspopup` is not fully implemented in browsers, care should be taken in using it. https://a11ysupport.io/tech/aria/aria-haspopup_attribute 

https://html5accessibility.com/stuff/2021/02/02/haspopup-haspoop/
